pkg/projects/Microsoft.NETCore.App/Microsoft.NETCore.App.pkgproj

```diff
@@ -41,13 +41,10 @@
       </Dependency>
     </ItemGroup>
 
-    <!-- Use NETStandard.Library version suffix for version suffix of Microsoft.NETCore.App -->
     <PropertyGroup>
-      <NETStandardBuildNumber Condition="'%(Dependency.Identity)' == 'NETSTandard.Library'">$([System.Text.RegularExpressions.Regex]::Match('%(Dependency.Version)', -.*))</NETStandardBuildNumber>
-      <VersionSuffix Condition="'$(NETStandardBuildNumber)' != ''">$(NETStandardBuildNumber)</VersionSuffix>
-      <VersionSuffix Condition="'$(CLIBuildVersion)' != ''">$(VersionSuffix)-$(CLIBuildVersion)</VersionSuffix>
+    <!-- For RC2  we need to start with -rc2-3 so that it is higher then the last published -rc2-23931 -->
+    <VersionSuffix>-rc2-3$(CLIBuildVersion)</VersionSuffix>
```

Comment:
Is there a way we can share this logic with https://github.com/dotnet/cli/blob/rel/1.0.0/scripts/dotnet-cli-build/Utils/BuildVersion.cs?

I'm going to need this version inside of our C# build scripts in order to replace the project.json.template files.  I can totally dupe it, but it would be nice if there was a way to have this defined in 1 spot.